web/templates/popovers/user_group_info_popover.hbs

```diff
@@ -12,12 +12,16 @@
             </div>
         </li>
         {{#if (or displayed_members.length displayed_subgroups.length)}}
-            <li role="none" class="popover-menu-list-item text-item italic">
-                {{#tr}}
-                {members_count, plural, =1 {1 member} other {# members}}
-                {{/tr}}
-            </li>
+            {{#if settings_data.user_can_access_all_other_users}}
```

Comment:
Did you test this? I don't think this will work as expected.

You would need to pass the value returned by calling this function.